command/genesis/utils.go

```diff
@@ -163,6 +164,55 @@ func parseBurnContractInfo(burnContractInfoRaw string) (*polybft.BurnContractInf
 	}, nil
 }
 
+type baseFeeInfo struct {
+	baseFee            uint64
+	baseFeeEM          uint64
+	baseFeeChangeDenom uint64
+}
+
+// parseBaseFeeConfig parses provided base fee configuration and returns baseFeeInfo
+func parseBaseFeeConfig(baseFeeConfigRaw string) (*baseFeeInfo, error) {
+	baseFeeInfo := &baseFeeInfo{
+		command.DefaultGenesisBaseFee,
+		command.DefaultGenesisBaseFeeEM,
+		command.DefaultGenesisBaseFeeChangeDenom,
+	}
+
+	baseFeeConfig := strings.Split(baseFeeConfigRaw, ":")
+	if len(baseFeeConfig) > 3 {
+		return baseFeeInfo, errors.New("invalid number of arguments for base fee configuration")
+	}
+
+	if baseFeeConfig[0] != "" {
```

Comment:
Prevent index out of range:

```suggestion
	if len(baseFeeConfig) >= 1 && baseFeeConfig[0] != "" {
```